vislib/include/vislib/sys/Log.h

```diff
@@ -196,6 +208,7 @@ namespace sys {
              * @param path The path to the physical log file
              * @param level The log level used for this target
              */
+            [[deprecated("use char* version instead")]]
```

Comment:
Why not just delete the function if it throws an error anyways? This way we can get rid of all calls to this function instead of finding out when MegaMol explodes.